mozdownload/scraper.py

```diff
@@ -743,6 +743,7 @@ def __init__(self, branch='mozilla-central', build_number=None, date=None,
         """Create instance of a tinderbox scraper."""
         self.branch = branch
         self.build_number = build_number
+        self.builds = []
```

Comment:
We do not initialize the list for other scrapers in `__init__`. So I don't think we have to do it here too. We could fix it for all in a separate PR, but that's low priority.

Reply:
I must have misunderstood you in one of the previous change requests... I thought this is what you asked me to do...
I will remove it again in the next commit...

Reply:
Oh I see. The comment about that I made Oct last year. So please forgive me that I simply forgot about it. Meanwhile I just want to see as much changes as necessary to actually get this feature working without regressing something else. If we want to initialize `self.builds` in `__init__` then we can/should do it for all scrapers at once in a different PR.